packages/caliper-samples/src/contract/scalardl/src/main/java/com/example/contract/simple/SimpleOpen.java

```diff
@@ -0,0 +1,26 @@
+package com.example.contract.simple;
```

Comment:
I left some feedbacks on this contract, but those are applicable to other contracts. So please update other contracts as well.